docs/rest_guidelines.md

```diff
@@ -121,8 +121,22 @@ The status codes are grouped in five classes:
 * 4xx client error – the request contains bad syntax or cannot be fulfilled
 * 5xx server error – the server failed to fulfill an apparently valid request
 
-Links to status code lists:
+In total there are over 60 possible HTTP Status codes, but APIs should focus on using a limited set in a consistent manner.
+
+The most important HTTP Status codes are: 
```

Comment:
Maybe use _commonly used_ instead of _important_?

Reply:
Agree, push a new commit for this